data.sql

```diff
@@ -1,22 +1,39 @@
 /* Populate database with sample data. */
```

Comment:
**Changes Required**
- [ ] Since the vet, specialization and visit tables don't exist, the project requirement is that you should have created them in the `schema.sql` and inserted into them from here. Let us please make these changes.